lib/src/over_react_test/jacket.dart

```diff
@@ -61,18 +61,26 @@ class TestJacket<T extends react.Component> {
   final Element mountNode;
   final bool attachedToDocument;
   final bool autoTearDown;
+  bool _isMounted;
```

Comment:
Can we initialize this to false for future-proofing?
```suggestion
  bool _isMounted = false;
```

Reply:
Good call. will do.